components/feature/downloads/src/test/java/mozilla/components/feature/downloads/AbstractFetchDownloadServiceTest.kt

```diff
@@ -1058,6 +1058,27 @@ class AbstractFetchDownloadServiceTest {
         verify(service).unregisterNotificationActionsReceiver()
     }
 
+    @Test
+    @Config(sdk = [Build.VERSION_CODES.P])
+    fun `WHEN a download is completed the file MUST be added to the download system database`() {
```

Comment:
I'm not sure if you want to rename this test since it's being configured for and checking a specific SDK version, and since there is different MediaStore approach for build versions > P.

Reply:
👍 
I updated the test to include that only apply on devices older that Q:
"WHEN a download is completed on devices older than Q the file MUST be added manually to the download system database"